CONTRIBUTING.md

```diff
@@ -6,3 +6,41 @@ We love your input! We want to make contributing to Roboflow Notebooks as easy a
 - When adding or editing notebooks, make sure to specify the versions of pip packages 📦 you are using.
 
 Please keep in mind that repository is still under construction 🚧. We will add more contribution guidelines soon.
```

Comment:
Let's drop that line

Reply:
just removed it, it'll be updated in the new PR